cmd/kubectl-testkube/config/config.go

```diff
@@ -10,7 +10,11 @@ var Config config
 func init() {
 	Config = config{}
 	l := log.DefaultLogger
-	err := Config.Init()
+
+	// set default analytics enabled
+	Config.Data.AnalyticsEnabled = true
+
+	err := Config.Init(Config.Data)
```

Comment:
It's a bit weird, Config methods have access to Data field, it's even public. Init and Save should have a parameter, if we plan to pass non Data field